tools/util/getDiff.ts

```diff
@@ -0,0 +1,43 @@
+import { execSync } from "child_process";
+import { basename, dirname } from "path";
+
+export type ValidEventName = "push" | "pull_request" | "uncommitted";
+
+/**
+ * Gets an array of all the changed folders using the git diff
+ * @returns {Promise<string[]>} Array of unique paths to the changed folders
+ */
```

Comment:
why are you so inconsistent? just jsdoc all functions if we are going to do it here